src/Microsoft.AspNet.OData.Shared/Common/SRResources.resx

```diff
@@ -988,4 +988,22 @@
   <data name="ShouldHaveNavigationPropertyInNavigationExpandPath" xml:space="preserve">
     <value>A navigation property expand path should have navigation property in the path.</value>
   </data>
+  <data name="ResourcesShouldbePresent" xml:space="preserve">
+    <value>ResourceSetWrapper should have ResourceWrappers in it</value>
+  </data>
+  <data name="ResourceSetWrapperSupported" xml:space="preserve">
+    <value>Can only add ResourceWrapper to ResourceSetWrapper</value>
+  </data>
+  <data name="ChangedObjectTypeMismatch" xml:space="preserve">
+    <value>Cannot use Changed Object of type '{0}' on an entity of type '{1}'.</value>
+  </data>
+  <data name="DataModificationException" xml:space="preserve">
+    <value>Core.DataModificationException</value>
+  </data>
+  <data name="ContentID" xml:space="preserve">
+    <value>Core.ContentID</value>
```

Comment:
Do we need a more relevant value here?

Reply:
This is as per design, as per Core Vocab annotations